packages/reports/src/Containers/progress-slider-stream.tsx

```diff
@@ -24,13 +31,7 @@ const ProgressSliderStream = ({ contract_info, is_loading, server_time }) => {
     );
 };
 
-ProgressSliderStream.propTypes = {
-    contract_info: PropTypes.object,
-    is_loading: PropTypes.bool,
-    server_time: PropTypes.object,
-};
-
-export default connect(({ common, portfolio }) => ({
+export default connect(({ common, portfolio }: any) => ({
```

Comment:
Could you import and use RootStore type here?
```suggestion
export default connect(({ common, portfolio }: RootStore) => ({
```

Reply:
RootStore is not yet migrated to `tsx`